R/dplyr_join.R

```diff
@@ -0,0 +1,95 @@
+#' Join Spark tbls.
+#'
+#' These functions are wrappers around their `dplyr` equivalents that set
+#' Spark SQL-compliant values for the `suffix` argument by replacing dots (`.`)
+#' with underscores (`_`). See [join] for a description of the general purpose
+#' of the functions.
+#'
+#' @inheritParams dbplyr::join.tbl_sql
+#'
+#' @name join.tbl_spark
+NULL
+
+#' @rdname join.tbl_spark
+#' @export
```

Comment:
Minor: currently the entry for this method is `export(inner_join.tbl_spark)` in the NAMESPACE file, which is a bit off from the truth.

I think if you make the suggested change below and re-run `roxygen2::roxygenize()`, the NAMESPACE entry will become `S3method(inner_join,tbl_spark)`, which is more technically correct.

```suggestion
#' @importFrom dplyr inner_join
#' @export
```

Reply:
Agreed, should be an s3 method. I made the changes - it required reexporting the generics from `dplyr` in order to register the s3 method (I tried using just the `importFrom` block as you suggested but that did not register the methods).